tools/topology/topology1/sof-apl-da7219.m4

```diff
@@ -157,21 +157,21 @@ PCM_PLAYBACK_ADD(HDMI3, 7, PIPELINE_PCM_7)
 # BE configurations - overrides config in ACPI if present
 #
 
-#SSP 5 (ID: 0) with 19.2 MHz mclk with MCLK_ID 0 (unused), 1.536 MHz blck
+#SSP 5 (ID: 0) with 19.2 MHz mclk with MCLK_ID 1 (unused), 1.536 MHz blck
```

Comment:
if it *has* to be 1, then it's probably not "unused?"

Reply:
GLK needs to be updated too in that case, since it also has to be 1 there (and is already)

Reply:
no, I think the comment was intentional. The MCLK is only relevant for the headset codec IIRC, it's not used by the amplifier.
and while in theory we could use 2 MCLKs for the two different SSPs, I am not sure this was ever tested.

Reply:
the topology uses SSP5 (speaker - maxim98357a, BE link 0) so this is correct and doesn't need a change.